Dockerfile

```diff
@@ -1,5 +0,0 @@
-FROM kong
```

Comment:
With this deleted, how should a developer work on the plugin?

Reply:
I'm curious how do other plugins handle this? I'm fine with whatever but consistency would be nice but that's a someday task probably :)